cfg/cfg.go

```diff
@@ -152,10 +173,35 @@ func Print() {
 // ReadConfig reads content of file by path cfg.FilePath,
 // parses its content as array of Che plugin meta objects and returns it.
 // If any error occurs during read, log.Fatal is called.
+//
+// Deprecated
 func ReadConfig() []model.PluginMeta {
+	raw := readConfigFile()
+
+	metas := make([]model.PluginMeta, 0)
+	if err := json.Unmarshal(raw, &metas); err != nil {
+		log.Fatal(err)
```

Comment:
Broker has special method for this purpose that insures that error message is sent to WS master. I see that we exit program here including in all codebase. 
Consider changing this file to return error when something happens and then main.go of brokers would be able to send error to WS master

Reply:
This is a bit more involved of a change so it's broken out into a separate commit: https://github.com/eclipse/che-plugin-broker/pull/40/commits/b76456c6409d05e29010e1d0489228d717213949

In short: some of the broker implementations didn't include `common.Broker`, so to allow `broker.PrintFatal` to be called, I had to change interfaces and structs. Let me know what you think.

Reply:
It is nice that we now send error to WS master properly. Though I didn't get why we need to rename  `PushEvents` function. And it is better to remove main.go in other brokers instead of extending their codebases.

Reply:
I decided to remove them separately as this is left over from my last PR rather then something your PR specifically needs

Reply:
Thanks for that -- I'll rebase onto current master.

Reply:
Regarding the `PushEvents` function, that is something I'm currently working on a decent solution to; in e.g. the Theia broker, we're defining a `Broker` interface that includes a `PushEvents` function. To be able to use a Theia broker's `PrintFatal` we need to include `common.Broker` in this interface, which causes a name collision on the `PushEvents` function. However, since we're no longer using the non-unified brokers standalone, this can likely be removed.